allolib-wasm/include/al_WebAudioVisualBridge.hpp

```
@@ -0,0 +1,384 @@
+        uint32_t _pad0;
+        uint32_t _pad1;
+    };
+    GPUUniformBuffer<FeatureParams> mFeatureParamsBuffer;
+    ComputeShader mFeatureShader;
+
+    GraphicsBackend* mBackend = nullptr;
+    int mFFTSize = 0;
+    int mSampleRate = 0;
+
+    // Beat detection state (CPU-side)
+    float mBeatHistory[64];
```

Clang (14.0.6)

[warning] do not declare C-style arrays, use std::array<> instead

(modernize-avoid-c-arrays)